.eslintrc.json

```diff
@@ -23,14 +23,15 @@
         "ecmaVersion": 2018
     },
     "rules": {
+        "semi": "error",
```

Comment:
indentation space here is 4, although in prettierrc configuration is set to 2 🧐

Reply:
Oups 🙊